src/Processor/LocaleProcessor.php

```diff
@@ -17,11 +17,12 @@ public function process(string $locale): string
         $locales = array_filter(Locales::getLocales(), function (string $targetLocale) use ($locale): bool {
             return
                 strpos($targetLocale, $locale) === 0 &&
-                strpos($targetLocale, '_') !== false
+                strpos($targetLocale, '_') !== false &&
+                strlen($targetLocale) === 5
             ;
         });
 
-        return $locales[array_key_first($locales)];
+        return preg_replace('/en_[A-Z]{2}/', 'en_US', $locales[array_key_first($locales)]);
```

Comment:
As this is a special case (I wonder how many more we will notice in the future? 😄), I would do it even before the `array_filter` with a simple:

```php
if ($locale === 'en') {
    return 'en_US';
}
```

And leave this one as it is. Probably the best thing would be to have a table of *default* locales (e.g. `['en' => 'en_US', 'fr' => 'fr_FR', 'it' => 'it_IT']` etc. But that's an idea for the future 🖖